.github/workflows/ci.yml

```diff
@@ -11,7 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: stefanoeb/eslint-action@1.0.2
```

Comment:
@bingbongle fyi reason for this - https://github.com/stefanoeb/eslint-action was deprecated a few days ago